modules/management-console-agent/src/main/java/org/apache/ignite/agent/ws/WebSocketManager.java

```diff
@@ -46,22 +48,25 @@
 import org.springframework.messaging.simp.stomp.StompSessionHandler;
 import org.springframework.util.MimeTypeUtils;
 import org.springframework.web.socket.WebSocketHttpHeaders;
-import org.springframework.web.socket.client.jetty.JettyWebSocketClient;
+import org.springframework.web.socket.client.standard.StandardWebSocketClient;
 import org.springframework.web.socket.messaging.WebSocketStompClient;
 
-import static java.net.Proxy.NO_PROXY;
-import static java.net.Proxy.Type.SOCKS;
-import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.concurrent.TimeUnit.SECONDS;
-import static java.util.stream.Collectors.toList;
 import static org.apache.ignite.agent.utils.AgentObjectMapperFactory.binaryMapper;
-import static org.apache.ignite.agent.utils.AgentUtils.EMPTY;
-import static org.eclipse.jetty.client.api.Authentication.ANY_REALM;
+import static org.glassfish.tyrus.client.ClientManager.createClient;
+import static org.glassfish.tyrus.client.ClientProperties.PROXY_URI;
+import static org.glassfish.tyrus.client.ClientProperties.SSL_ENGINE_CONFIGURATOR;
+import static org.glassfish.tyrus.client.ThreadPoolConfig.defaultConfig;
+import static org.glassfish.tyrus.container.grizzly.client.GrizzlyClientProperties.SELECTOR_THREAD_POOL_CONFIG;
+import static org.springframework.util.Base64Utils.encodeToString;
 
 /**
  * Web socket manager.
  */
 public class WebSocketManager extends GridProcessorAdapter {
+    /** Empty string array. */
+    private static final String[] EMPTY = new String[0];
```

Comment:
Lets move this constant to common utilities, such constant required in many places usually.